FireCARES/firecares
# Add EMS_ENABLED env variable

## About

This PR adds a environment variable boolean for enabling/disabling EMS Heatmaps.

## Relevant Issues

- Closes https://github.com/FireCARES/firecares/issues/510

## Features
Provides environment variable `EMS_ENABLED` or change variable default inside `settings/base.py`
When True, should try to load EMS Heatmap from URL

<img width="448" alt="Screen Shot 2019-04-08 at 10 08 09 AM" src="https://user-images.githubusercontent.com/4162671/55735141-66967f80-59e6-11e9-883c-6a9cc4af4c30.png">
<img width="916" alt="Screen Shot 2019-04-08 at 10 07 46 AM" src="https://user-images.githubusercontent.com/4162671/55735160-6dbd8d80-59e6-11e9-9084-221f5c4bf76c.png">

When False, it should skip all EMS Heatmap

<img width="438" alt="Screen Shot 2019-04-08 at 10 08 18 AM" src="https://user-images.githubusercontent.com/4162671/55735203-829a2100-59e6-11e9-81cc-16d1e71a91c2.png">
<img width="911" alt="Screen Shot 2019-04-08 at 10 08 53 AM" src="https://user-images.githubusercontent.com/4162671/55735217-87f76b80-59e6-11e9-9daf-1d444875a284.png">

## Testing Instructions

1. Go to `settings/base.py` and change `EMS_ENABLED` to True.
2. Check that EMS Heatmaps are being shown : http://192.168.33.15:8000/departments/88369/mehlville-fire-protection-district
(Not all departments have EMS data) 
3. Go to `settings/base.py` and change `EMS_ENABLED` to False
4. Go to http://192.168.33.15:8000/departments/88369/mehlville-fire-protection-district and check that EMS Heatmap option is not shown.
